src/sentry/static/sentry/app/views/discover/missingProjectWarningModal.tsx

```diff
@@ -31,18 +30,18 @@ export default class MissingProjectWarningModal extends React.Component<
            have access to.`);
 
     return (
-      <Modal show onHide={() => {}}>
```

Comment:
Does this still get shown as a modal? Does the caller need to include a wrapping `Modal`?

Reply:
it was actually already calling this modal with `openModal`

Reply:
https://github.com/getsentry/sentry/blob/6edc3a39576dc15d51114aa5683c1fbf03ea6f16/src/sentry/static/sentry/app/views/discover/queryBuilder.tsx#L384-L391